packages/ibm-products-styles/src/__tests__/__snapshots__/styles.test.js.snap

```diff
@@ -2126,10 +2126,9 @@ p.c4p--about-modal__copyright-text:first-child {
   display: grid;
   height: 100%;
   color: var(--cds-text-primary, #161616);
-  grid-template-rows: minmax(auto, 100%);
 }
 .c4p--create-full-page:has(.c4p--create-full-page__header) {
-  grid-template-rows: auto minmax(auto, 100%);
+  grid-template-rows: auto minmax(min-content, 100%);
```

Comment:
Hey @baraaabdelsalam ,
Looks like there is a height/scroll issue with this change. At smaller viewport heights, the footer buttons section need to be fixed and inner content has to scrollable which is not happening now. 
![image](https://github.com/user-attachments/assets/d3c9ad84-5c03-441d-a49d-f3edb8e103d7)